tools/indexer/pkg/config/config.go

```diff
@@ -1,21 +1,40 @@
 package config
```

Comment:
I dont really think a separate pkg is needed for this. I dont mind having this but it seems unnecessary just to have a struct definition and default values in separate pkg